From ce4db296b2b39153b2435c4fd945a3b62b2f474f Mon Sep 17 00:00:00 2001
From: Ghennadi Procopciuc <ghennadi.procopciuc@nxp.com>
Date: Tue, 17 May 2022 10:00:18 +0300
Subject: [PATCH 15/30] s32cc: Remove the generation of u-boot-s32.bin

u-boot-s32.bin isn't needed anymore as ATF can use u-boot-nodtb.bin

Issue: ALB-8823
Upstream-Status: Pending 

Signed-off-by: Ghennadi Procopciuc <ghennadi.procopciuc@nxp.com>
Signed-off-by: Zhantao Tang <zhantao.tang@windriver.com>
---
 Makefile | 5 +----
 1 file changed, 1 insertion(+), 4 deletions(-)

diff --git a/Makefile b/Makefile
index a90e094359..49e5ea6956 100644
--- a/Makefile
+++ b/Makefile
@@ -927,7 +927,7 @@ ALL-y += u-boot-rockchip.bin
 endif
 
 ifeq ($(CONFIG_NXP_S32CC),y)
-ALL-y += u-boot-s32.bin u-boot-s32.cfgout
+ALL-y += u-boot-s32.cfgout
 endif
 
 LDFLAGS_u-boot += $(LDFLAGS_FINAL)
@@ -1177,9 +1177,6 @@ endif
 ifeq ($(CONFIG_NXP_S32CC),y)
 %.cfgout: FORCE
 	$(Q)$(MAKE) $(build)=arch/arm/mach-s32/s32-cc $@
-
-u-boot-s32.bin: u-boot.dtb u-boot-nodtb.bin
-	$(call if_changed,binman)
 endif
 
 quiet_cmd_copy = COPY    $@
-- 
2.17.1

